WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -80,14 +80,16 @@ - (RemoteUser *)remoteUserFromDictionary:(NSDictionary *)dictionary
 
 - (NSArray *)remoteBlogsFromJSONArray:(NSArray *)jsonBlogs
 {
-    NSMutableArray *remoteBlogs = [NSMutableArray arrayWithCapacity:[jsonBlogs count]];
-    for (NSDictionary *jsonBlog in jsonBlogs) {
-        BOOL isJetpack = [jsonBlog[@"jetpack"] boolValue];
-        if (!isJetpack || JetpackREST.enabled) {
-            [remoteBlogs addObject:[self remoteBlogFromJSONDictionary:jsonBlog]];
-        }
+    NSArray *blogs = jsonBlogs;
+    if (!JetpackREST.enabled) {
```

Comment:
Ohhh snap!! See what i mean?. This coffee ain't no good anymore. Sorry about the noise sir! i missed the assignment in line 83!!!